devnotes/release-process.md

```diff
@@ -0,0 +1,60 @@
+# GitHub oriented Swift package release process
+
+Release process:
+
+Check out the latest code, or at the mark you want to release.
+While the CI system is pretty solid, it's worthwhile to do a fresh repository clone, run a full build, and all the relevant tests before proceeding.
+
+The process will be tagging, but not until we've made a commit that explicitly sets up the downloadable packages from GitHub release artifacts.
+
+Steps:
+
+- run `./scripts/build-xcframework.sh` to build the XCFramework and regenerate all the relevant C/FFI library components.
+- Capture the build hash (sha256) of the XCFramework that the script prints out at the end of it's process.
+It will look something like:
+
+```
+SHA256(yniffiFFI.xcframework.zip)= 9aa2dd069662613b66749a257d753fc7007afe4817278edfd6cc902de94b5f6c
+```
+
+- switch the binary target in Package.swift for yniffiFFI to a url and checksum reference. The url references the download path for release artifacts from a tag.
```

Comment:
To be honest, this is the only place that I didn't quite get right away. What do you think if add something like "desired result" and include a code snippet of resulting `Package.swift` portion:
```swift
        .binaryTarget(
            name: "yniffiFFI",
            // Uncomment `path`, and comment out url and checksum, for local development
            // path: "./lib/yniffiFFI.xcframework"
            url: "https://github.com/heckj/yswift/releases/download/0.1.1/yniffiFFI.xcframework.zip",
            checksum: "9aa2dd069662613b66749a257d753fc7007afe4817278edfd6cc902de94b5f6c"
        ),
```